<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="sourceLabel">
         <property name="text">
          <string>Source:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sourceLineEdit">
         <property name="text">
          <string>rtsp://admin:123Dmv321@192.168.0.99:554/Streaming/Channels/101/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="fpsLabel">
           <property name="text">
            <string>FPS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fpsLineEdit">
           <property name="text">
            <string>25</string>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>591</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Device commands</string>
    </property>
    <widget class="QPushButton" name="enableVKBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable VK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disableVKBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable VK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enableTestBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable testing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disableTestBtn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable testing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="speed0Btn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="speed1Btn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="speed2Btn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextStepBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next step -&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prevStepBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;- Prev step</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextPatternBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next pattern -&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prevPatternBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;- Prev pattern</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brightBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set brightness</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="brightLineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>101</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
    <widget class="QPushButton" name="obeleziBtn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>obelezi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="praviMaskuBtn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Pravi Masku</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
